assignments/arrays.js

```diff
@@ -63,34 +63,67 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a 2011 Jeep Wrangler` );
+
+for (i = 0; i < inventory.length; i++) {
+console.log(`${inventory[i].car_model} was made by ${inventory[i].car_make} in ${inventory[i].car_year}`);
+}
 
```

Comment:
You're looping through the whole array, and returning that string for each car in the array. Instead, we only want to log one string for car 33. You were close. Just add a conditional statement. As we're looping through all that data, if we come across a car with id 33, plug the values of car 33 into this string and return it. 

```suggestion

 for (i = 0; i < inventory.length; i++) {
  if (inventory[i].id === 33) {
    console.log(`${inventory[i].car_model} was made by ${inventory[i].car_make} in 
    ${inventory[i].car_year}`);
  };
};

```